src/components/Card.js

```diff
@@ -51,13 +43,23 @@ export default class Card {
     //get the card view
     this._cardImageEl = this._cardElement.querySelector(".card__image");
     this._cardTitleEl = this._cardElement.querySelector(".cards__title");
-    
+
     this._cardImageEl.src = this._link;
     this._cardImageEl.alt = this._name + " " + "Image";
     this._cardTitleEl.textContent = this._name;
-    //set event listeners
-    this._setEventListeners();
-    //return the card
-    return this._cardElement;
+    this._likeButton = this._cardElement.querySelector(".card__like-button");
+    this._deleteButton = this._cardElement.querySelector(
+      ".card__delete-button"
+    );
+        //set event listeners
+        if (this.isLiked) {
```

Comment:
Needs correction: the code for hanging the like button state gets duplicated. Please create a method and use it here and when updating the isLiked state

```js
_renderLikes() {
  if (this._isLiked) {
    // add active class to the like button
  } else {
    // remove active class from the like button
  }
```

Reply:
I created a method and called it in getview(). Thank you!